### PR TITLE
workaround for acm hub CRDs apply.

### PR DESCRIPTION
**Pull request by morvencao**
Signed-off-by: morvencao <lcao@redhat.com>

### CODE REVIEW

File: doc/hypershift/hypershift-bm.md [hunk 1]
```diff
@@ -114,44 +114,29 @@ oc wait --for=condition=ManagedClusterConditionAvailable managedcluster/${HYPERS
...
+  ```
+  ```bash
+  API_SERVICE_NODEPORT=$(oc --kubeconfig=${HYPERSHIFT_MGMT_CLUSTER_KUBECONFIG} -n ${HYPERSHIFT_HOSTING_NAMESPACE}-${HYPERSHIFT_DEPLOYMENT_NAME} get svc/kube-apiserver -o jsonpath='{.spec.ports[?(@.port==6443)].nodePort}')
+  oc --kubeconfig=${HYPERSHIFT_MGMT_CLUSTER_KUBECONFIG} -n ${HYPERSHIFT_HOSTING_NAMESPACE}-${HYPERSHIFT_DEPLOYMENT_NAME} port-forward svc/kube-apiserver ${API_SERVICE_NODEPORT}:6443 --address=${LOCAL_MACHINE_IP} &
   ```
 
-7. Check the ACM addons(policy and application) are available:
+  _Note_: If you want to import a managed cluster to the HyperShift hosted cluster without external access, you have to make sure the managed cluster can access the local machine that executes the commands above.
+
+6. Apply the ACM Hub CRDs to the hypershift hosted cluster(workaround because application addon doesn't support hosted mode currently):
```
Comment (clyang82) [inline doc/hypershift/hypershift-bm.md:137]: Do not mention it is a workaround. that is current solution for release-2.6.

Comment (morvencao) [reply doc/hypershift/hypershift-bm.md]: As we discussed offline, currently we create deliver a job with manifestwork to create the hub CRDs, so we don't need this any more.

Comment (morvencao) [reply doc/hypershift/hypershift-bm.md]: the guide is much simpler now.

**Pull request merged**